### PR TITLE
Amit/home

### PR DESCRIPTION
**Pull request by Amit-Aditaya**
Bottom Nav UI complete (testing another PR)

### CODE REVIEW

**Comment by Amit-Aditaya** [comment]
custom app bar created, home page background image set

**Comment by Amit-Aditaya** [comment]
testing pushing branch from forked repo to original repo

File: lib/features/tabs/screens/tab_screen.dart [hunk 1]
```diff
@@ -1,56 +1,89 @@
...
-      _selectedIndex = index;
-    });
-  }
-
+class TabScreen extends HookWidget {
   static const List<Widget> _pages = [
     HomeScreen(),
     FavoriteScreen(),
     CartScreen(),
+    LoginScreen(),
```
Comment (rifat0153) [inline lib/features/tabs/screens/tab_screen.dart:14]: Profile screen will be here. When user is not logged in they will be redirected to Login. Will explain to you later.

**Review by rifat0153 — CHANGES_REQUESTED** [review]
Review the comments and make changes accordingly. For now, make a dummy profile view. Then make a new commit. PR will still remain the same. I will merge it afterward.

**Pull request closed without merging**